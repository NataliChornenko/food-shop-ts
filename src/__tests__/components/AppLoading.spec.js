import React from 'react'
import { shallow } from 'enzyme'
import AppLoading from 'components/AppLoading'

describe('app loadig', () => {
  const appLoading = shallow(<AppLoading />)

  it('renders properly', () => {
    expect(appLoading).toMatchSnapshot()
  })
})
