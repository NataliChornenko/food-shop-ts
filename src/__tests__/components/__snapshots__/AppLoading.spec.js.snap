// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Name of the group renders properly 1`] = `ShallowWrapper {}`;

exports[`app loadig renders properly 1`] = `ShallowWrapper {}`;
